abpframework/abp
# Remove abp package . it cause local ref problem in ABP

### Description

Resolves #16034

When packages installed via npm.js test read from node_modules so it cause the local ref problem in Testing. So I have removed abp angular packages in package.json in ng-packs. it fixed . 

### Checklist


### How to test it?
- go to abp/npm/ng-packs
- remove node_modules (if exists)
-  run yarn install
- Go to a test file for example `validation-utils.spec.ts`
- change a dependency outside of that package like `config-state.service.ts`
expected: the test should detect and work with local changes.
